src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -840,7 +840,7 @@ def wrapper(*vpath, **params):
         return wrapper
 
     @staticmethod
-    def Resource(method=None, path=None, status=None, query_params=None):
+    def Resource(method=None, path=None, status=None, query_params=None):  # noqa: N802
```

Comment:
If we want to keep these (`Resource`, `Proxy`, etc.), though they're in violation of PEP naming guidelines (`assertTaskResult` is at least aligned with the `unittests` free-spirit naming), [pep8-naming allows](https://github.com/PyCQA/pep8-naming#options): 

```ini
# tox.ini
[flake8]
ignore-names = Resource, Collection, ...
# Not sure if we still need to add the defaults: setUp,tearDown,setUpClass,tearDownClass,setUpTestData,failureException,longMessage,maxDiff
```

Reply:
I suggest to keep the ``noqa`` comments in the ``controllers/__init__.py`` file because when using them in ``ignore-names`` we might oversee warning where they are useful. So we can isolate the exceptions to only those files where we WANT/NEED to break the PEP naming guidelines.